pocs/site_location_details/__init__.py

```diff
@@ -0,0 +1,75 @@
+from astroplan import Observer
```

Comment:
This is an awkard name for a directory and also kind of just an awkward file. All of this is really just setting up configuration details and shouldn't really be set up as a module.

Could move this file into `pocs/utils/location.py` for now although we might want to change that in the long run.